.github/actions/markPullRequestsAsDeployed/index.js

```diff
@@ -18,13 +18,29 @@ const isProd = JSON.parse(
 );
 const token = core.getInput('GITHUB_TOKEN', {required: true});
 const date = new Date();
-const message = `🚀 Deployed 🚀 to ${
-    isProd ? 'production' : 'staging'
-} on ${date.toDateString()} at ${date.toTimeString()}`;
 
 const octokit = github.getOctokit(token);
 const githubUtils = new GithubUtils(octokit);
 
+function getDeployTableMessage(platformResult) {
+    const emoji = platformResult === 'success' ? '✅' : '❌';
+    return platformResult + emoji;
+}
+
+const androidResult = getDeployTableMessage(core.getInput('ANDROID', {required: true}));
+const desktopResult = getDeployTableMessage(core.getInput('DESKTOP', {required: true}));
+const iOSResult = getDeployTableMessage(core.getInput('IOS', {required: true}));
+const webResult = getDeployTableMessage(core.getInput('WEB', {required: true}));
+
+const workflowURL = `${process.env.GITHUB_SERVER_URL}/${process.env.GITHUB_REPOSITORY}
+    /actions/runs/${process.env.GITHUB_RUN_ID}`;
+
+let message = `🚀 [Deployed](${workflowURL}) 🚀 to 
+    ${isProd ? 'production' : 'staging'} on ${date.toDateString()} at ${date.toTimeString()}`;
+
+message += `\n\n platform | result \n ---|--- \n android|${androidResult} \n desktop|${desktopResult}`;
+message += `\n iOS|${iOSResult} \n web|${webResult}`;
+
```

Comment:
NAB: Curious if it's possible/worth DRY-ing up these blocks of code?

Reply:
Which lines specifically did you have in mind? The four instances of `getDeployTableMessage`? I could loop through them and store them in a map? Did you have any other ideas?

Reply:
Maybe there's something I'm missing but looks to me that these lines are the same in the `markPullRequestsAsDeployed.js`, just wondering if we could have this logic in some sort of helper function?

Reply:
Maybe there's something I'm missing but looks to me that these lines are the same in the `markPullRequestsAsDeployed.js`, just wondering if we could have this logic in some sort of helper function?

Reply:
Ahh - Sorry this is confusing, but `index.js` is just a compiled JS version for GitHub actions with all the libraries, etc needed to run. So yes it will be identical to the source code (which is not `index.js`). So you can just ignore all the code in index.js as it was compiled via `npm run gh-actions-build`